README.md

```diff
@@ -1,6 +1,199 @@
+platform linux -- Python 3.9.7, pytest-6.1.2, py-1.11.0, pluggy-0.13.1 -- /usr/bin/python3
+cachedir: .pytest_cache
+rootdir: /src/libs, configfile: pytest.ini
+plugins: tap-3.3, tavern-1.12.2
+collected 1 item
+
+../libs/test_example_functional.tavern.yaml::Verify the service status resource PASSED                                                           [100%]
+
+```
+
+1. To run the smoke tests which will invoke a simple http response code checker trigger the entrypoint script to run the smoke tests.
+2. Pass in the `example_smoke.json` file with `-f` option
+3. pass in the overloaded URL with `-u`
+
+```
+/src/app $ entrypoint.sh smoke -f /src/libs/example_smoke.json -u http://httpbin.org
+Running smoke tests...
+2022-03-28 20:04:27,901 suite_name: example_smoke_tests
+2022-03-28 20:04:27,901 file_path: /src/libs/example_smoke.json
+2022-03-28 20:04:27,901 Running example_smoke_tests...
+2022-03-28 20:04:27,901 default_url: http://httpbin.org
+2022-03-28 20:04:27,901 override_url: http://httpbin.org
+2022-03-28 20:04:27,902 configured_url: http://httpbin/
+2022-03-28 20:04:27,902 Testing {"path": "/get?color=blue&size=big", "expected_status_code": 200, "method": "GET", "body": null, "headers": {}, "url": "http://httpbin.org/get?color=blue&size=big"}
+2022-03-28 20:04:27,932 Starting new HTTP connection (1): httpbin.org:80
+2022-03-28 20:04:28,031 http://httpbin.org:80 "GET /get?color=blue&size=big HTTP/1.1" 200 1682
+2022-03-28 20:04:28,032 Testing {"path": "/post", "expected_status_code": 200, "method": "POST", "body": "HI THERE!", "headers": {"accept": "application/json", "custom-header": "just like this"}, "url": "http://httpbin.org/post"}
+2022-03-28 20:04:28,069 Starting new HTTP connection (1): httpbin.org:80
+2022-03-28 20:04:28,147 http://httpbin.org:80 "POST /post HTTP/1.1" 200 1745
+2022-03-28 20:04:28,149 Testing {"path": "get?color=blue&size=big///", "expected_status_code": 200, "method": "GET", "body": null, "headers": {}, "url": "http://httpbin.org/get?color=blue&size=big///"}
+2022-03-28 20:04:28,185 Starting new HTTP connection (1): httpbin.org:80
+2022-03-28 20:04:28,261 http://httpbin.org:80 "GET /get?color=blue&size=big/// HTTP/1.1" 200 1694
+2022-03-28 20:04:28,262 Testing {"path": "get?badreturncode=true", "expected_status_code": 200, "method": "GET", "body": null, "headers": {}, "url": "http://httpbin.org/get?badreturncode=true"}
+2022-03-28 20:04:28,300 Starting new HTTP connection (1): httpbin.org:80
+2022-03-28 20:04:28,375 http://httpbin.org:80 "GET /get?badreturncode=true HTTP/1.1" 200 1669
+2022-03-28 20:04:28,376 MAIN_ERRORS = 0
+2022-03-28 20:04:28,377 PASS: example_smoke_tests
+2022-03-28 20:04:28,377 passed!
+2022-03-28 20:04:28,377 PASS: example_smoke_tests passed!
+/src/app $
+```
+
+1. You will most likely use the `/src/libs/tavern_global_config.yaml` file with the functional tests (that invoke pytest + tavern)
```

Comment:
What does (that invoke pytest + tavern) mean here?

Reply:
pytest is a tool; tavern is a framework